### ISSUE TITLE
Fortran lexer, array definition syntax error

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#594]
(Original issue 887 created by zmiimz on 2013-07-17T11:08:32.840823+00:00)

Since fortran 2003 an array constructor may be bracketed with square brackets, [ ], instead of (/ /).. Currently it is highlighted as an error.

```
#!fortran

program test
integer :: m1(6)
integer :: m2(6)
m1 = [ 1, 0, 0, 0, 5, 0 ]
m2 = (/ 1, 0, 0, 0, 5, 0 /)
end program test
```

**Comment by Anteru** [comment]
(Original issue 887 last updated on 2014-01-09T17:57:07.760866+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by kyuubi1692 on 2013-10-18T11:04:24.115017+00:00)

Possible Solution : [887](https://bitbucket.org/kyuubi1692/kyuubi-pygments-main/commits/44466d1cd209b9fa4713104302375d48e3d12205)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T17:56:51.487735+00:00)

Closes #887: allow brackets in fortran.

→ <<cset ccbd47c664a2>>

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2014-01-09T17:57:07.774399+00:00)

see ccbd47c664a2
